src/System.Private.CoreLib/src/System/Threading/Tasks/TPLETWProvider.cs

```diff
@@ -219,13 +221,14 @@ public class Keywords // thisname is important for EventSource
          Level = EventLevel.Informational, Keywords = Keywords.TaskTransfer | Keywords.Tasks)]
         public void TaskScheduled(
             int OriginatingTaskSchedulerID, int OriginatingTaskID,  // PFX_COMMON_EVENT_HEADER
-            int TaskID, int CreatingTaskID, int TaskCreationOptions, int appDomain)
+            int TaskID, int CreatingTaskID, int TaskCreationOptions)
```

Comment:
EventSource has special logic that defines the logging contract based on the argument declarations.   Thus by removing and argument you have changed the contract with readers of this log (which is what we are trying to avoid).   

Thus the way to avoid any compat concerns is to leave the AppDomain argument (it can be a default argument).

But frankly, I think the way it is right now (where GetDomainID simply returns a constant), is fine).